tests/tasks/start_mock_wifi.yml

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Ensure NetworkManager is running
+  service:
+    name: NetworkManager
+    state: started
+
+- name: Load mac80211_hwsim kernel module to mock a wifi network
+  shell: modprobe mac80211_hwsim radio=2 && sleep 5
+
+- name: Restart NetworkManager and wpa_supplicant
+  service:
+    name: "{{ item }}"
+    state: restarted
+  with_items:
+    - NetworkManager
+    - wpa_supplicant
+
+- name: Configure wlan0 and wlan1 (mock wifi interfaces)
+  shell: |
+    ip link set up wlan0
+    ip link set up wlan1
+    nmcli device set wlan1 managed off
+    ip add add 203.0.113.1/24 dev wlan1
```

Comment:
This command is correct, use `ip addr add 203.0.113.1/24 dev wlan1` instead. Thanks!

Reply:
> Thanks for the efforts, please could you add a detailed description on the commit message? Thanks!

Detailed description added, thanks.

Reply:
> ip addr add 203.0.113.1/24 dev wlan1

Changed into `ip addr add 203.0.113.1/24 dev wlan1`, thanks.